internal/operands/common-templates/common-templates-v0.12.2.yaml

```diff
@@ -0,0 +1,10949 @@
+# Version v0.12.2
```

Comment:
Can you move this file outside of the source code directory? Maybe to a `data` directory? The source code directories will probably not be included in the operator container image.